Core/scripts/dirac-install.py

```diff
@@ -843,11 +843,17 @@ def downloadAndExtractTarball( tarsURL, pkgName, pkgVer, checkHash = True, cache
   tarName = "%s-%s.tar.gz" % ( pkgName, pkgVer )
   tarPath = os.path.join( cliParams.targetPath, tarName )
   tarFileURL = "%s/%s" % ( tarsURL, tarName )
+  tarFileCVMFS = "/cvmfs/dirac.egi.eu/installSource/%s" % tarName
```

Comment:
I would use /cvmfs/ilc.desy.de to put both dirac and my own tarballs. That should be available everywhere I run, and I don't have to depend on another repo. So it would be great if this where configurable.